data-plane/core/pom.xml

```diff
@@ -181,7 +181,7 @@
 
     <dependency>
       <groupId>io.micrometer</groupId>
-      <artifactId>micrometer-registry-prometheus</artifactId>
+      <artifactId>micrometer-registry-prometheus-simpleclient</artifactId>
```

Comment:
We use the simpleclient, since that is _backwards_ compatible to the old APIs.

While `vertx-micrometer` 4.5.10 is _still_ on the `1.12` APIs, we do inherit the `1.13` APIs via the Quarkus BOM... 
And hence we really need the simpleclient, we are using that - to avoid nasty breakages